windows/src/desktop/kmshell/locale/kan-Knda-IN/locale.xml

```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE Locale SYSTEM "../../xml/localedef.dtd">
 <Locale Version="10.0.1100.0" OnlineProductID="30">
   <String Type="PlainText" Id="S_LocaleAuthors">www.ಸಿರಿಗನ್ನಡ.com (www.sirigannada.com)</String>
-  <String Type="PlainText" Id="SK_UIFontName">Noto Sans Kannada</String>
+  <String Type="PlainText" Id="SK_UIFontName">Calibri, &quot;Noto Sans Kannada&quot;</String>

```

Comment:
This is not going to work everywhere; the `SK_UIFontName` property is used within Delphi code, which doesn't support a comma-separated list of fonts, for example https://github.com/keymanapp/keyman/blob/c3acee9b554b7e8241c64af360c26fbceec2c055/windows/src/engine/keyman/Menu_KeyboardItems.pas#L91

Reply:
I have removed Calibri.
Since the "Noto Sans Kannada" has space in its name, should we retain "&quot;" before and after the font name?
If not I will remove it.